src/App.js

```diff
@@ -1,12 +1,52 @@
 import React from "react";
 import "./App.css";
+import {useState} from 'react';
 
 function App () {
-    return (
-      <main>
-        <p>React State Lab</p>
-      </main>
-    );
+  const [counter, setCounter] = useState(0);
+  //counter = current score 
+  const [accumulator, setAccumulator] = useState(1);
+ // useState says prep for user interaction
+  const countClick = () => setCounter(counter + accumulator);
+// eventlistener when user clicks
+  function  setCount () {
+      if (counter < 10){
+        window.alert("You can't afford that!")
+        return 
+      }
+      setAccumulator (accumulator + 1)
+      setCounter (counter - 10)
+    }
+
+function resetGame() {
+  setCounter(0)
+  setAccumulator(1)
 }
 
+let gameInPlay = (
+  // a variable holding a function
+  <div>
+      <h1> Current Score: {counter} </h1>
+      <button onClick={countClick}>+{accumulator}</button>
+      <button onClick={setCount}>Pay 10 points to change from +{accumulator} </button>
```

Comment:
The sentence here doesn't match what the test is looking for which is why you are failing that one test.